tests/Http/TagsTest.php

```diff
@@ -27,7 +30,7 @@ public function testAddingATagToAPost()
         $this->assertResponseStatus(200);
 
         // Make sure that the post's tags are updated.
-        $this->assertContains('Good Photo', $post->tagNames());
+        $this->assertContains('Good Photo', $post->tags()->pluck('tag_name'));
```

Comment:
Since we do same thing so often, it might be nice to make a `tagNames()` method on the Post using the same implementation.